java.base/src/main/java/java/lang/ProcessEnvironment.java

```diff
@@ -61,8 +62,8 @@
 final class ProcessEnvironment {
     private ProcessEnvironment() {}
 
-    static final EnvironmentMap theEnvironment;
-    static final Map<String, String> theUnmodifiableEnvironment;
+    @SerializeAsZero
```

Comment:
As an aside, we probably should automatically apply the equivalent of `@SerializeAsZero` for any field which is replaced at run time if we aren't doing so already.

Reply:
Yep,  we are.  Anything that is @RuntimeAspect gets serialized as zero.